src/analysis/normed_space/conformal_linear_map.lean

```diff
@@ -112,6 +112,16 @@ lemma injective {f' : M →L[R] N} (h : is_conformal_map f') : function.injectiv
 let ⟨c, hc, li, hf'⟩ := h in by simp only [hf', pi.smul_def];
   exact (smul_left_injective _ hc).comp li.injective
 
+lemma ne_zero [h : nontrivial M] {f' : M →L[R] N} (hf' : is_conformal_map f') :
+  f' ≠ 0 :=
+begin
+  intros w,
+  rcases nontrivial_iff.mp h with ⟨a, b, hab⟩,
```

Comment:
Rather than naming the typeclass hypothesis as `h`, I think you can leave it unnamed, and use the lemma `exists_ne` to get an `a` not equal to zero.